playbooks/roles/thoth-ssh-key-check/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
checking and creating the key is a good idea, but it should go into the secret via `oc create secret` rather than the secret template, see comment at https://github.com/thoth-station/kebechet/pull/11/files#diff-1c1f52855a99250874e1c80fe5f5acd0R54

Reply:
Made the required change :+1: